decomp/semantics/uds.py

```diff
@@ -202,9 +227,18 @@ def add_annotation(self, annotation: 'UDSDataset') -> None:
             the annotations to add to the graphs in the corpus
         """
 
+        # If this is a RawUDSDataset, we need to tell the graph which properties
+        # feature raw annotations, as we need to be able to prevent querying
+        # on raw properties.
+        raw_properties = annotation.raw_properties \
```

Comment:
Once this sort of assignment breaks lines, it's usually more readable to just use a normal if-then.